proxy/HostStatus.h

```diff
@@ -46,7 +49,14 @@ struct HostStatRec_t {
   unsigned int down_time; // number of seconds that the host should be down, 0 is indefinately
 };
 
-const std::string stat_prefix = "host_status.";
+static const std::string stat_prefix = "proxy.process.host_status.";
+static const std::string MANUAL      = "manual";
```

Comment:
Why don't you define all the tree and use them in other places?

Reply:
I refactored this and am using a struct now, see latest commit.